### PR TITLE
Add https service handler

### PR DESCRIPTION
**Pull request by matthewbentley**

### CODE REVIEW

**Comment by keshavdv** [comment]
lgtm

**Comment by EvanKrall** [comment]
Looks good, would be nice to add a test though.

**Comment by EvanKrall** [comment]
This could work for the tests:

```diff
diff --git a/tests/test_checker.py b/tests/test_checker.py
index ee0aca7..209b463 100644
--- a/tests/test_checker.py
+++ b/tests/test_checker.py
@@ -68,7 +68,7 @@ class TestChecker(TestCase):
             is_up_patch.assert_called_once_with(se.name, port=se.port)


-class TestHTTPChecker(tornado.testing.AsyncHTTPTestCase):
+class BaseTestHTTPHTTPSChecker(object):

     def get_app(self):
         return tornado.web.Application([
@@ -129,6 +129,14 @@ class TestHTTPChecker(tornado.testing.AsyncHTTPTestCase):
         self.assertEqual(400, response[0])


+class TestHTTPChecker(BaseTestHTTPHTTPSChecker, tornado.testing.AsyncHTTPTestCase):
+    pass
+
+
+class TestHTTPSChecker(BaseTestHTTPHTTPSChecker, tornado.testing.AsyncHTTPTestCase):
+    pass
+
+
 class TestServer(tornado.tcpserver.TCPServer):
     @tornado.gen.coroutine
     def handle_stream(self, stream, address):
```

**Comment by EvanKrall** [comment]
er, yeah, AsyncHTTPSTestCase for the second one, and parameterize checker.check_http I guess.

File: setup.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 setup(
     name="hacheck",
-    version="0.9.0",
+    version="0.10.0",
```
Comment (keshavdv) [inline setup.py]: Also bump the version in __init__.py

**Pull request merged**